L1Trigger/L1TGlobal/src/CorrCondition.cc

```diff
@@ -646,13 +659,15 @@ const bool l1t::CorrCondition::evaluateCondition(const int bxEval) const {
           phiIndex1 = (candMuVec->at(cond1bx, obj1Index))->hwPhiAtVtx();  //(*candMuVec)[obj0Index]->phiIndex();
           etaIndex1 = (candMuVec->at(cond1bx, obj1Index))->hwEtaAtVtx();
           etIndex1 = (candMuVec->at(cond1bx, obj1Index))->hwPt();
+	  uptIndex1 = (candMuVec->at(cond1bx, obj1Index))->hwPtUnconstrained(); // Added by R.Cavanaugh for displaced muons
           chrg1 = (candMuVec->at(cond1bx, obj1Index))->hwCharge();
           etaBin1 = etaIndex1;
           if (etaBin1 < 0)
             etaBin1 = m_gtScales->getMUScales().etaBins.size() + etaBin1;
           //		   LogDebug("L1TGlobal") << "Muon phi" << phiIndex1 << " eta " << etaIndex1 << " etaBin1 = " << etaBin1  << " et " << etIndex1 << std::endl;
 
           etBin1 = etIndex1;
+	  uptBin1 = uptIndex1;   // Added by R.Cavanaugh for displaced muons
```

Comment:
Please remove meaningless comments.  Git versioning system keeps track of authors of commits.

Reply:
@rekovic 
Many thanks for the helpful request to remove comments.  Done.